scss/_base_links.scss

```diff
@@ -0,0 +1,13 @@
+// Base styling for links
+@mixin vf-b-links {
+  a {
```

Comment:
No :hover or :visited styling has been provided so far and I've already raised this with @yaili as a separate issue in #448